src/libs/OptionsListUtils.js

```diff
@@ -516,10 +529,8 @@ function createOption(accountIDs, personalDetails, report, reportActions = {}, {
         const lastMessageTextFromReport = getLastMessageTextForReport(report);
         const lastActorDetails = personalDetailMap[report.lastActorAccountID] || null;
         const lastActorDisplayName =
-            hasMultipleParticipants && lastActorDetails && lastActorDetails.accountID !== currentUserAccountID
-                ? lastActorDetails.firstName || PersonalDetailsUtils.getDisplayNameOrDefault(lastActorDetails)
-                : '';
-        let lastMessageText = lastActorDisplayName ? `${lastActorDisplayName}: ${lastMessageTextFromReport}` : lastMessageTextFromReport;
+            hasMultipleParticipants && lastActorDetails && lastActorDetails.accountID !== currentUserAccountID ? lastActorDetails.firstName || lastActorDetails.displayName : '';
```

Comment:
Why don't we use `PersonalDetailsUtils.getDisplayNameOrDefault(lastActorDetails)` here anymore?

Reply:
ok, thanks for pointing this one out - restoring this method